### ISSUE TITLE
After updating from Sublime Text 4126 -> 4142 the yaml format is all yellow

### ISSUE THREAD

**Issue by arianf** [Monokai/monokai-pro-sublime-text#159]
After updating from Sublime Text 4126 -> 4142 the yaml syntax highlighting is the same color for keys and string values.

<img width="1247" alt="Screenshot 2022-11-10 at 9 44 23 AM" src="https://user-images.githubusercontent.com/7397857/201168544-7e0764d6-4798-49f7-8038-8393f9286a33.png">

**Comment by arianf** [comment]
In the release notes for Build 4142:
* Added syntax-based code folding
* Various syntax highlighting improvements

**Comment by arianf** [comment]
My config:

```
	"theme": "Monokai Pro (Filter Spectrum).sublime-theme",
	"color_scheme": "Monokai Pro (Filter Spectrum).sublime-color-scheme",
```

**Comment by brandonchinn178** [comment]
Seems like the new build changed the labels. e.g. YAML keys now have the scope:
```
source.yaml meta.mapping.key.yaml meta.string.yaml string.unquoted.plain.out.yaml
```
which Monokai Pro doesn't have any color definitions for

**Comment by brandonchinn178** [comment]
As a workaround, go to "Customize Color Scheme" and add:
```json5
    {
      // https://github.com/Monokai/monokai-pro-sublime-text/issues/159
      "name": "YAML strings",
      "scope": "source.yaml meta.mapping.key string",
      "foreground": "#ff6188",
    },
```

**Comment by Monokai** [comment]
Fixed in 1.2.0.
